awscli/customizations/s3/utils.py

```diff
@@ -406,19 +406,39 @@ def uni_print(statement, out_file=None):
     out_file.flush()
 
 
-def bytes_print(statement):
+def bytes_print(statement, stdout=None):
```

Comment:
I guess what I was meaning why not make sys.stdout the default then? You have the:
```py
if stdout is None:
    stdout = sys.stdout
```
and sys.stdout is not mutable so it should be fine as:
```py
def bytes_print(statement, stdout=sys.stdout):
```